src/pages/OrganizationDetail.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { withOidcSecure } from '@axa-fr/react-oidc';
+import { PanelGrid, PanelGridItem } from '../components/PanelGrid';
```

Comment:
This should be imported after the `pdc-api` imports, along with other component imports.